src/smc-webapp/projects.cjsx

```diff
@@ -525,7 +525,8 @@ class ProjectsStore extends Store
             return 0
         others = []
         for i in v
-            if not i.deleted and (show_hidden or not i.get('users').get(account_id).get('hide'))
+            # deleted projects have a map node " 'deleted': true' standard projects do not have this property
+            if (not (i.get('deleted') == true )) and (show_hidden or not i.get('users').get(account_id).get('hide'))
```

Comment:
The only actual non-comment request I made was related to this line, but you seem not to have seen that.